### ISSUE TITLE
Automatic decline mails are not sent out

### ISSUE THREAD

**Issue by danielneu** [amzn/amazon-payments-magento-plugin#193]
Hi,

it seems the automatic sending of decline mails from the cron stopped working (eu branch).
I can see the orders going to on hold, which means the job is running, but I do not receive mails for this. I do receive other mails, like new order or new registrations. If i press on "sync with Amazon" the mail is sent out.

Please have a look at it,
Daniel

**Comment by danielneu** [comment]
Hi,

I'm still not receiving the mails via cron. Only after pressing the sync with amazon button.

Best wishes
Daniel

**Comment by jonahellison** [comment]
Hmm, I'm receiving emails from the cron. Is the order being synced with the cronjob? Magento also has its own internal cron, sometimes truncating the cron_schedule table resets it.

**Comment by danielneu** [comment]
Yes,

the order is updated, I will try the truncating fix and see if it works.
